src/test/java/org/jabref/logic/crawler/StudyDatabaseToFetcherConverterTest.java

```diff
@@ -60,9 +60,10 @@ public void getActiveFetcherInstances() throws Exception {
         StudyDatabaseToFetcherConverter converter = new StudyDatabaseToFetcherConverter(studyRepository.getActiveLibraryEntries(), importFormatPreferences);
         List<SearchBasedFetcher> result = converter.getActiveFetchers();
 
-        Assertions.assertEquals(2, result.size());
+        Assertions.assertEquals(3, result.size());
         Assertions.assertEquals(result.get(0).getName(), "Springer");
         Assertions.assertEquals(result.get(1).getName(), "ArXiv");
+        Assertions.assertEquals(result.get(2).getName(), "Medline/PubMed");
```

Comment:
In general, it is better you compare two collections directly, this makes it easier to spot differences in case of failure.
e.g. you have on the one hand:
expectedFetcherNames = LIst.of(.......)
actualFetcherNames = result.stream.map(Fetcher::getName).toCollection...
assertEquals(expectedFetcherNames,